### PR TITLE
More style guide additions

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/contributing/style-guide.md [hunk 1]
```diff
@@ -151,7 +151,32 @@ Here's an example that encompasses all the above styles: If you are not parsing
 
 ## Code Blocks
 
-Use code blocks to format scripts, such as the JSON example below. This is important for scripts and CLI. Always use these to format programming language scripts (i.e., SQL for Sumo queries, JSON for logs). Format blocks of code by placing triple backticks before and after the code.
+Use code blocks to format scripts, such as the JSON example below. This is important for scripts and CLI. Format blocks of code by placing triple backticks before and after the code.
+
+If you know the code language, include that in the first set of backticks to activate syntax highlighting. See [this list](https://prismjs.com/#supported-languages) of supported languages.  Always use these to format programming language scripts (i.e., `sql` for Sumo queries, `json` for logs).
```
Comment (kimsauce) [inline docs/contributing/style-guide.md:156]:
```suggestion
If you know the code language, include that in the first set of backticks to activate syntax highlighting. See [this list](https://prismjs.com/#supported-languages) of supported languages. Always use these to format programming language scripts (i.e., `sql` for Sumo queries, `json` for logs).
```

File: docs/contributing/style-guide.md [hunk 2]
```diff
@@ -106,7 +150,9 @@ For example, the first time you use AWS Application Load Balancer (ALB), you int
 
 ## Beta Releases
 
-For a Closed Beta release, you'll need to exclude it from the nav and search engine results so that users will need the link to access it:
+### Closed Beta
```
Comment (jpipkin1) [inline docs/contributing/style-guide.md:153]: Can you add a sentence that describes what a closed beta is?

File: docs/contributing/style-guide.md [hunk 3]
```diff
@@ -125,15 +171,28 @@ For a Closed Beta release, you'll need to exclude it from the nav and search eng
   ```
 1. Do _not_ add the doc to `sidebars.ts`.
 1. Publish the doc.
-1. When the feature is moved from Beta to GA, remove the Robots meta tag and Beta label, and add the doc to sidebars.ts.
 
+When the feature moves from Beta to GA, remove the Robots meta tag, remove the Beta label, and add the doc to sidebars.ts.
+
+### Open Beta
```
Comment (jpipkin1) [inline docs/contributing/style-guide.md:177]: Can you add a sentence that describes what an open beta is?

**Review by jpipkin1 — DISMISSED** [review]
Very nice updates, @kimsauce. I only had a couple of comments about beta releases.

File: docs/contributing/style-guide.md (hunk 3 above)
Comment (kimsauce) [reply docs/contributing/style-guide.md:177]: Thanks, @jpipkin1 ! I'll create an Asana ticket for defining open/closed beta.

File: docs/contributing/style-guide.md (hunk 2 above)
Comment (kimsauce) [reply docs/contributing/style-guide.md:153]: I'll create an Asana ticket for defining open/closed beta.

**Pull request merged**